docs/how_to_guides/creating_and_editing_expectations/how_to_create_expectations_while_exploring_data_in_a_notebook_without_using_a_data_context.rst

```diff
@@ -3,16 +3,227 @@
 How to create Expectations while exploring data in a notebook, without using a Data Context.
 ============================================================================================
 
-.. admonition:: Admonition from Mr. Dickens
+Building :ref:`Expectations` as you conduct exploratory data analysis is a great way to ensure that your insights about data processes and pipelines remain part of your team's knowledge.
 
-    "Whether I shall turn out to be the hero of my own life, or whether that station will be held by anybody else, these pages must show."
+This guide will help you get started quickly in Great Expectations, without even setting up a :ref:`Data Context`. All you need is a notebook and some data.
 
+.. admonition:: Prerequisites: This how-to guide assumes you have already:
 
-This guide is a stub. We all know that it will be useful, but no one has made time to write it yet.
+    - Installed Great Expectations (e.g. ``pip install great_expectations``)
+    - Have access to a notebook (e.g. ``jupyter notebook``, ``jupyter lab``, etc.)
+    - Be able to access data from your notebook
+    - Nothing else. Unlike most how-to guides, these instructions do *not* assume that you have configured a Data Context by running ``great_expectations init``.
```

Comment:
```suggestion
```

Reply:
I very much want to keep this item.

Once we have three or four of them, I'm planning to link them from (and to) a "magic free" section in the Quick Start tutorial that will allow users to get a taste of Great Expectations without even having to set up a Data Context.

Of course, a big part of that tutorial will be selling users on the value of Data Contexts, too.